core/server/models/session.js

```diff
@@ -0,0 +1,39 @@
+const ghostBookshelf = require('./base');
+
+let Session,
```

Comment:
either multiple `let` or define as `const` inline

Reply:
The whole code base is full of 

```
let X;

X = fn...
```

>  define as const inline

I like that. If you like it too, then let's use this 😁